watchdog/src/time.rs

```diff
@@ -0,0 +1,49 @@
+use std::time::Duration;
+
+#[cfg(not(target_arch = "wasm32"))]
+use std::cell::RefCell;
+
+#[cfg(not(target_arch = "wasm32"))]
+use std::time::Instant;
+
+#[cfg(not(target_arch = "wasm32"))]
+thread_local! {
+    /// Canister inner uptime since start in nanoseconds.
+    static START: RefCell<Instant> = RefCell::new(Instant::now());
+}
```

Comment:
nit: similar to the comment about listing all the endpoints in `main.rs`, it's in general a good practice to keep all of the canister's state in one place, to make it easy for the reader to see. This is documented in the "Effective Rust Canisters" document: https://mmapped.blog/posts/01-effective-rust-canisters.html#clear-state